devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -184,11 +180,10 @@ def configure() {
 
 	// temperature minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
-	def manufacturer = device.getDataValue("manufacturer")
-	if (manufacturer == "Heiman"|| manufacturer == "HEIMAN") {
+	if (isHeimanEF()) {
 		return refresh() +
 		        zigbee.temperatureConfig(30, 300) +
-		        zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 21600, 0x10) +
+		        zigbee.addBinding(0x0001) +
```

Comment:
It's usually best to configure reporting for every attribute, even if it is automatically configured for reporting by the device. Is there a reason that wasn't done here?

Reply:
I've discussed with the manufacture engineer this morning, and according to them, for periodically battery report, I need to bind Power Configuration cluster 0x0001, and in my test the battery will report every 2 hours after binding 0x0001.

Reply:
I'll check more about the battery value

Reply:
the `addBinding` message is generated inside the `configureReporting` function and `zigbee.POWER_CONFIGURATION_CLUSTER` is `0x0001`, so it seems like it's probably best to leave this as-is.

Reply:
> the `addBinding` message is generated inside the `configureReporting` function and `zigbee.POWER_CONFIGURATION_CLUSTER` is `0x0001`, so it seems like it's probably best to leave this as-is.

ok, I'll make new PR for the device